drivers/cc110x/cc110x_communication.c

```diff
@@ -113,3 +113,41 @@ uint8_t cc110x_status(cc110x_t *dev)
     } while (status != cc110x_cmd(dev, CC110X_STROBE_STATUS));
     return status;
 }
+
+void cc110x_wakeup_from_rx(cc110x_t *dev)
+{
+    if (dev->state != CC110X_STATE_RX_MODE ||
+        dev->state != CC110X_STATE_RECEIVING) {
+        return;
+    }
+
+    cc110x_cmd(dev, CC110X_STROBE_IDLE);
+    dev->state = CC110X_STATE_IDLE;
+}
+
+void cc110x_power_off(cc110x_t *dev)
+{
+    /* Ensure device is on IDLE state */
+    cc110x_wakeup_from_rx(dev);
+
+    /*
+     * Datasheet page 9 table 4.
+     *
+     * To achieve the lowest power consumption GDO's must
+     * be programmed to 0x2F
+     */
+    cc110x_write(dev, CC110X_REG_IOCFG2, CC110X_GDO_CONSTANT_LOW);
+    cc110x_write(dev, CC110X_REG_IOCFG1, CC110X_GDO_CONSTANT_LOW);
+    cc110x_write(dev, CC110X_REG_IOCFG0, CC110X_GDO_CONSTANT_LOW);
+
+    /* go to sleep */
+    cc110x_cmd(dev, CC110X_STROBE_OFF);
+
+    /*
+     * Set CS pin high so the device gets de-selected, and thus
+     * the sleep mode transition takes place
```

Comment:
This already happens in `cc110x_cmd()` which calls `spi_transfer_bytes()` which de-selects the CS GPIO at the end.

Reply:
Oh right, I had it on a previous version, will remove it.